README.md

```diff
@@ -241,6 +243,15 @@ docker run -e 'PROXY_SERVICE_HOST=http://stackexchange.com' \
            quay.io/ukhomeofficedigital/nginx-proxy:v1.0.0
 ```
 
+The example below will return "404" for the URL /notfound.
+```shell
+docker run -e 'PROXY_SERVICE_HOST=http://stackexchange.com' \
+           -e 'PROXY_SERVICE_PORT=80' \
+           -e 'ADD_NGINX_HTTP_CFG=proxy_cache_path /data/nginx/cache levels=1:2 keys_zone=static:10m;' \
+           -p 8443:443 \
+           quay.io/ukhomeofficedigital/nginx-proxy:v1.0.0
+```
```

Comment:
Why will it do that? Because the endpoint doesn't exist on the upstream server?
I can't see anything in the config relating to /not found.
Sorry if I'm being dense!